src/stats/inc/BayesianJointPdf.h

```diff
@@ -97,6 +97,11 @@ class BayesianJointPdf : public BaseJointPdf<V,M> {
 
   //@}
 
+  //! @name Using declarations
+  //@{
+  using BaseScalarFunction<V, M>::lnValue;
+  //@}
```

Comment:
Yes and no.  Though, technically yes.  Here's the deal:

If the user didn't call `lnValue` in their application code, then it's fine.  QUESO interacts with a `BaseScalarFunction` type for run-time polymorphism, and in that class everything is declared so no `using` declaration is needed.

If the user calls `lnValue` (for whatever reason) in their application code, then they'll need to add a `using` declaration to prevent the derived class from hiding all the other `lnValue` names declared in the base class.

Typically the user doesn't call `lnValue`.  QUESO does.

Will this fix the breakage?
1.  Add a super class to `BaseScalarFunction`, call it `A`;
2.  Move all the declarations of `lnValue` from `BaseScalarFunction` to `A`;
3.  Add a `using` declaration to `BaseScalarFunction` to reveal all the `lnValue` names;
4.  Hope.

I can try this locally to see if it works.